S24-testing/15-done-testing.t

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env raku
```

Comment:
I was to focused on the test body and overlooked this line. It has to be omitted from tests because they're not ran as scripts.